### PR TITLE
hardening/777_documents_about_environment_test

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue https://github.com/telefonicaid/fiware-cygnus/issues/777
* No test are required
* Assignee @frbattid

### CODE REVIEW

**Comment by frbattid** [comment]
As explained in the issue, this file must be under `test/e2e/`.

File: doc/user_and_programmer_guide/test_environment.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+##Introduction
+
+This is a private environment. All the information about it is described in a document in Google Docs where we will update all the changes. The document has restricted access (Contact us for more information):
```
Comment (frbattid) [inline doc/user_and_programmer_guide/test_environment.md]: This is a private environment for e2e testing addressed to fiware-cygnus developers.

Comment (frbattid) [inline doc/user_and_programmer_guide/test_environment.md]: Contact --> contact

Comment (pcoello25) [inline doc/user_and_programmer_guide/test_environment.md]: Fixed in 0a6814a

**Comment by frbattid** [comment]
LGTM

**Pull request merged**